src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2398,3 +2398,56 @@
 type: group
 short-summary: Manage log analytics workspace for a vm.
 """
+
+helps['vm monitor metrics'] = """
+type: group
+short-summary: Manage metrics for a vm.
+"""
+
+helps['vm monitor metrics tail'] = """
+type: command
+short-summary: List the metric values for a VM.
+parameters:
+  - name: --aggregation
+    short-summary: The list of aggregation types (space-separated) to retrieve.
+    populator-commands:
+      - az vm monitor metrics list-definitions
+  - name: --interval
+    short-summary: >
+        The interval over which to aggregate metrics, in ##h##m format.
+  - name: --filter
+    short-summary: A string used to reduce the set of metric data returned. eg. "BlobType eq '*'"
+    long-summary: 'For a full list of filters, see the filter string reference at https://docs.microsoft.com/rest/api/monitor/metrics/list'
+  - name: --metadata
+    short-summary: Returns the metadata values instead of metric data
+  - name: --dimension
+    short-summary: The list of dimensions (space-separated) the metrics are queried into.
+    populator-commands:
+      - az vm monitor metrics list-definitions
+  - name: --namespace
+    short-summary: Namespace to query metric definitions for.
+    populator-commands:
+      - az vm monitor metrics list-definitions
+  - name: --offset
+    short-summary: >
+        Time offset of the query range, in ##d##h format.
+    long-summary: >
+        Can be used with either --start-time or --end-time. If used with --start-time, then
+        the end time will be calculated by adding the offset. If used with --end-time (default), then
+        the start time will be calculated by subtracting the offset. If --start-time and --end-time are
+        provided, then --offset will be ignored.
+  - name: --metrics
+    short-summary: >
+        Space-separated list of metric names to retrieve.
+    populator-commands:
+      - az vm monitor metrics list-definitions
+examples:
+  - name: List a VM's CPU usage for the past hour
```

Comment:
What about `List CPU usage of VM for past one hour`?

Reply:
one more example